core/templates/dev/head/exploration_editor/ExplorationEditor.js

```diff
@@ -931,7 +923,17 @@ oppia.controller('ExplorationSaveAndPublishButtons', [
 
         modalInstance.result.then(function(commitMessage) {
           _modalIsOpen = false;
-          saveDraftToBackend(commitMessage);
+          saveDraftToBackend(commitMessage, function() {
```

Comment:
Agreed. It's not needed here since the server is already in place to check for this!